newapi/ooniapi/rate_limit_quotas.py

```diff
@@ -59,78 +114,66 @@ def __init__(
             self._whitelisted_ipaddrs.add(ipaddress.ip_address(ipa))
```

Comment:
It's not part of the newly added code, but based on the type hints for 
```
        unmetered_pages=Optional[List[str]],
```

It can be `null`, yet there is no check for the value being null in the for loop.